storage/remote/write_handler.go

```diff
@@ -126,7 +126,7 @@ func (h *writeHandler) write(ctx context.Context, req *prompb.WriteRequest) (err
 
 		for _, hp := range ts.Histograms {
 			hs := HistogramProtoToHistogram(hp)
-			_, err = app.AppendHistogram(0, labels, hp.Timestamp, hs)
+			_, err = app.AppendHistogram(0, labels, hp.Timestamp, hs, nil)
```

Comment:
@codesome do we need something here similar for FloatHistograms?

Reply:
Yes, that is one of the future works.